### ISSUE TITLE
Accented characters in author name shown incorrectly when paged

### ISSUE THREAD

**Issue by kisslas** [git-for-windows/git#1001]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?

```
$ git --version --build-options
git version 2.11.0.windows.1
sizeof-long: 4
machine: x86_64
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

Reproduced both on 64-bit Win7 and 64-bit Win10. Failed to reproduce on both after downgrading to git version 2.10.2.windows.1. => Operating system very likely not a factor.

 - What options did you set as part of the installation? Or did you choose the
   defaults?

Using different options on the two systems used to reproduce. => Options very likely not a factor.

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

LANG is en_US.utf8; changing it to various other values makes no difference. LESSCHARSET is unset and setting it to various different values does not fix the issue (although the characters printed in the place of the accented characters do change).

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

Reproducible using git-bash (although found using a Cygwin terminal - but it would be much more complicated to clone that setup).

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
git config --global user.name "László"
git config --global core.pager less
git commit --allow-empty -m"dummy"
git --no-pager log -1
git log -1
```
 - What did you expect to occur after running these commands?

The last two commands should produce identical output containing the author name, `László`, with the correct accented characters.

 - What actually happened instead?

The penultimate command works as expected. The last command prints `L<C3><A1>szl<C3><B3>` in place of the user name, with the `<hex codes>` in reverse video.

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

Reproducible with any repository.

**Comment by PhilipOakley** [comment]
I think this is the same as #981 (and others). there is a test release that has fixed it for me (see #981).

**Comment by kisslas** [comment]
Oops, I saw #981, but somehow got confused about what versions it applies to.
I have now tested with `git version 2.10.2.windows.1.895.g8126884` and can confirm that the problem does not appear in this version.
